sed/config/NXmpes_config-HEXTOF.json

```diff
@@ -159,25 +150,27 @@
     "frequency/@units": "kHz",
     "associated_beam": "/entry/instrument/beam_pump"
   },
-  "/ENTRY[entry]/INSTRUMENT[instrument]/beam_TYPE[beam_pump]": {
+  "/ENTRY[entry]/INSTRUMENT[instrument]/beamTYPE[beam_pump]": {
     "distance": 0.0,
     "distance/@units": "mm",
     "incident_wavelength": "@attrs:metadata/scientificMetadata/Laser/wavelength/value",
     "incident_wavelength/@units": "@attrs:metadata/scientificMetadata/Laser/wavelength/unit",
-    "pulse_duration": "@attrs:metadata/scientificMetadata/Laser/wavelength/pulse duration/value",
-    "pulse_duration/@units": "@attrs:metadata/scientificMetadata/Laser/wavelength/pulse duration/unit",
-    "incident_polarization": "@attrs:metadata/instrument/beam/pump/incident_polarization",
+    "incident_energy": "@attrs:metadata/scientificMetadata/Laser/energy/value",
+    "incident_energy/@units": "@attrs:metadata/scientificMetadata/Laser/energy/unit",
+    "pulse_duration": "@attrs:metadata/scientificMetadata/Laser/pulse duration/value",
+    "pulse_duration/@units": "@attrs:metadata/scientificMetadata/Laser/pulse duration/unit",
+    "incident_polarization": "@attrs:metadata/scientificMetadata/Laser/polarization",
```

Comment:
Convert into stokes vector (what is the given number anyways?)

Reply:
This is number is angle of QWP